ysh/cpython.py

```diff
@@ -176,18 +173,15 @@ def _ValueToPyObj(val):
 
         elif case(value_e.Slice):
             val = cast(value.Slice, UP_val)
-            step = 1
-            if val.step:
-                step = val.step.i
 
             if val.lower and val.upper:
-                return slice(val.lower.i, val.upper.i, step)
+                return slice(val.lower.i, val.upper.i)
```

Comment:
It might be OK to remove convertions fo/from value.Slice,  because we are only converting at the "edges" now

and I don't think any functions actually take a slice, and none should in the future either


We could remove it later too

Reply:
Ah right. The evaluator already operates on `value.Slice` in the subscript code. Sweet